service/runner.cc

```diff
@@ -33,6 +33,7 @@
 #include "soa/types/basic_value_descriptions.h"
 
 #include <future>
+#include <boost/filesystem.hpp>
```

Comment:
Let's avoid adding a dependency on boost/filesystem from here. getcwd is simple, works well, and is part of the C++ api.